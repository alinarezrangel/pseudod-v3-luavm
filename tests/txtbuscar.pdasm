PDVM 1.0
PLATFORM "pdcrt"

SECTION "code"
  ICONST 0
  LCONST 2
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 3
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 4
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 5
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 6
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 7
  LCONST 1
  MSG 0, 2, 1
  PRN
  NL

  ICONST 0
  LCONST 3
  LCONST 8
  MSG 0, 2, 1
  PRN
  NL
  ICONST 1
  LCONST 3
  LCONST 8
  MSG 0, 2, 1
  PRN
  NL
  ICONST 8
  LCONST 3
  LCONST 8
  MSG 0, 2, 1
  PRN
  NL
  ICONST 15
  LCONST 3
  LCONST 8
  MSG 0, 2, 1
  PRN
  NL

  ICONST 9
  LCONST 7
  LCONST 8
  MSG 0, 2, 1
  PRN
  NL
ENDSECTION

SECTION "procedures"
ENDSECTION

SECTION "constant pool"
  #0 STRING "buscar"
  #1 STRING "hola mundo"
  #2 STRING "la m"
  #3 STRING "h"
  #4 STRING "undo"
  #5 STRING "doque"
  #6 STRING "no"
  #7 STRING ""
  #8 STRING "haaa aahaa aaahaaa"
ENDSECTION

