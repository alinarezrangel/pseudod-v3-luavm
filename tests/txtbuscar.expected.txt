2
0
6
-1
-1
0
0
7
14
-1
9
